### ISSUE TITLE
Global name 'data' is not defined

### ISSUE THREAD

**Issue by dwhenson** [TheSavior/ESLint-Formatter#78]
I've tried installing this but I get the following error when I try to run: "Global name 'data' is not defined". I imagine I'm doing something very simple wrong, any advise would be appreciated.

**Comment by Niekes** [comment]
I have the same issue after updating:

```json
// ...
        "ESLint-Formatter": {
            "format_on_save": true,
            "local_eslint_path": {
                "osx": "/usr/local/bin/eslint_d"
            },
            "format_on_save_extensions": [
                "js",
                "jsx",
                "es",
                "es6",
                "vue",
                "babel"
            ]
        },
// ...
```

**Comment by Niekes** [comment]
If you use `eslint_d` you can set `"fix_to_stdout"` In the settings to `true`

That fixed my problem.

**Comment by yanqiangsjz** [comment]
I have the same issue, what's wrong?

**Comment by dwhenson** [comment]
I'm just using the standard eslint plugin, what's the difference with eslint_d?

**Comment by skeggse** [comment]
My sincerest apologies; this was an oversight on my part. I've released a fix in 2.4.1, which should be available just as soon as Package Control finds the new tag and adds it to https://packagecontrol.io/channel_v3.json.
